lib/index.js

```diff
@@ -18,43 +18,72 @@
 
 'use strict';
 
-function node_require(module) {
+console.log("Up to date");
+
+// Prevent React Native packager from seeing modules required with this
+function nodeRequire(module) {
     return require(module);
 }
 
-// If process is defined, we're running in node.
-function isNode() {
-    return typeof process == 'object' && (('' + process) == '[object process]' || typeof jest == 'object')
-}
 
-// function isElectronRenderer() {
-//     return isNode() && process.type === 'renderer'
-// }
+function getContext() {
+    if (typeof process !== 'undefined') {
+        if(typeof process === 'object' && process + '' === '[object process]') {
```

Comment:
What about collapsing this to simply the inner condition? The outer one doesn't give any additional value.